### DISCUSSION TITLE
Blitzwolf SHP10 sockets and 3kW electric heater

### DISCUSSION THREAD

**Discussion by Th0maz** [arendst/Tasmota#20647] [Q&A] [answered]
Hey everyone,
not a Tasmota related question, but more a hardware and experience question.
I've got a couple of Blitzwolf SHP10 plugs with power metering lying around and thought about using it for a 3kW electric heater.
The plug can supposedly handle up to 16A and the heater draws around 13A.
Did anyone ever use those devices under "heavy load"?
I want to heat a room, but not set it on fire ;)
Since the plug can measure its internal temperature, is there some kind of safety over-temperature power-off functionality built into tasmota?
Thanks
Thomas

**Comment by barbudor** [comment, marked as answer]
I always considered the "chinese" rating of those devices slighly over estimated but without any real proof
I wouldn't hook more than half of the 16A to such device

Still I have a Nous or a Gosund plug, given for 16A, that burned while only having a Wifi router connected on it. So sometime you cannot predict.

For an 3KW device I would look at something with a little more muscle, either a POW 20A, or a DIN power relay which would be driven by a tasmotized device.

**Reply by sfromis** [reply]
To me, it also depends on how often you want to switch the relay, as arching can reduce lifetime if the relay has an "optimistic rating".
